### DISCUSSION TITLE
keyboard.type missing commad key

### DISCUSSION THREAD

**Discussion by fischgeek** [johnlindquist/kit#1159] [Q&A] [answered]
Is it just me, or is the mac's command key missing from the enum?

**Comment by johnlindquist** [comment, marked as answer]
@fischgeek It's "LeftSuper":

```js
await keyboard.pressKey(Key.LeftSuper, Key.A)
await keyboard.releaseKey(Key.LeftSuper, Key.A)
```

They're mapped to the enums of the nutjs lib which which then presses/types them out.

**Reply by fischgeek** [reply]
Thanks! I'm a native Windows user so I didn't realize that. My bad.

**Reply by johnlindquist** [reply]
@fischgeek No worries at all, Key naming is inconsistent across every Keyboard-based project in existence. It's super annoying.
